docs/integration_test_job.md

```diff
@@ -25,7 +23,14 @@ workflows:
 
 ## Parameters
 
-These optional parameters allow configuring the test.
+- [common parameters](common.md#parameters) shared in all jobs.
+- [test-dir](#attach_workspace) (required string)
+- [env-file](#env-file) (optional string, default="")
+- [kind-config](#kind-config) (optional string, default="")
+- [kubernetes-version](#kubernetes-version) (optional string, default="v1.16.3")
+- [setup-script](#setup-script) (optional string, default="")
+- [test-dir](#test-dir) (required string)
+- [test-timeout](#test-timeout) (optional string, default="20m")
```

Comment:
@rossf7 I changed the structure of the docs a bit. Let me know if you don't like it :D

Reply:
Fine with me. Thx!